contrib/perl-razor2-client-agent/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "perl-razor2-client-agent"
+pkgver = "2.86"
+pkgrel = 0
+build_style = "perl_module"
+hostmakedepends = [
+    "gmake",
+    "perl",
+]
+makedepends = ["perl"]
+depends = ["perl", "perl-uri"]
+pkgdesc = "Distributed, collaborative spam detection and filtering network"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Artistic-2.0"
+url = "https://razor.sourceforge.net/index.html"
```

Comment:
website is dead?

Reply:
works for me, redirects to https://sourceforge.net/projects/razor/

Reply:
gets me a white page

Reply:
I changed to the redirect address https://sourceforge.net/projects/razor